src/Database/Dialect/PostgresDialectTrait.php

```diff
@@ -129,6 +129,24 @@ protected function _transformFunctionExpression(FunctionExpression $expression)
             case 'NOW':
                 $expression->name('LOCALTIMESTAMP')->add([' 0 ' => 'literal']);
                 break;
+            case 'DATE_ADD':
+                $expression
+                    ->name('')
+                    ->type(' + INTERVAL')
+                    ->iterateParts(function ($p, $key) {
+                        if ($key === 1) {
+                            $p = sprintf("'%s'", $p);
```

Comment:
Ahh yeah, I need to revert this, sorry, it is not possible to bound this parameter, because this must be part of the query ... `created + INTERVAL $1` is not correct syntax. However I've added check that the number must be numeric actually.... But it is up to developer to check whether the `hour` is valid SQL expression.